src/OSPSuite.Presentation/DTO/SensitivityAnalyses/SensitivityParameterDTO.cs

```diff
@@ -13,21 +13,25 @@ public SensitivityParameterDTO(SensitivityParameter sensitivityParameter) : base
          SensitivityParameter = sensitivityParameter;
       }
 
-      public string Name => SensitivityParameter.Name;
```

Comment:
For some reason, rename of parameter was not working (anymore). I also fixed that issue at the same time